.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+**What type of PR is this?**
+<!-- Bug, Chore, Documentation, Feature -->
+
+**What this PR does/ why we need it**:
+<!-- Explain, without going into the details, what this PR does, and what problem it solves. -->
+
+**Which issue(s) this PR fixes**:
+<!-- Add a link to the JIRA issue. Otherwise, put "no issue." -->
+
+**Special notes for your reviewer**:
```

Comment:
Im actually not sure what should go here @dlipovetsky what was the motivation do you remember, can we add a comment here please.

Reply:
I shamelessly copied this from the kubernetes PR template :smile: 

I have used it in the past to point out technical debt deliberately introduced by the PR.

Reply:
I don't know that a comment would help... this section might just be a place for anything that doesn't fit in any other section